lib/backup/azremote/azblob.go

```diff
@@ -383,3 +420,28 @@ func (fs *FS) ReadFile(filePath string) ([]byte, error) {
 
 	return b, nil
 }
+
+// envLookuper is an interface for looking up environment variables. It is
+// needed to allow unit tests to provide alternate values since the envtemplate
+// package uses a singleton to read all environment variables into memory at
+// init time.
+type envLookuper interface {
```

Comment:
I would suggest to simplify `envLookuper` to just a func:
```
type envLookuper func(name string) (string, bool)
```
This still allows to perform proper testing, but simplifies code a bit by removing an interface abstraction since interface contains just one method anyway.

Reply:
5085381c0